src/locale/nn.js

```diff
@@ -9,7 +11,7 @@ export default moment.defineLocale('nn', {
     monthsShort : 'jan_feb_mar_apr_mai_jun_jul_aug_sep_okt_nov_des'.split('_'),
     weekdays : 'sundag_måndag_tysdag_onsdag_torsdag_fredag_laurdag'.split('_'),
     weekdaysShort : 'sun_mån_tys_ons_tor_fre_lau'.split('_'),
```

Comment:
As you can see, I haven't touched nynorsk to make two letter versions, but we should.

Reply:
I agree. on that. Maybe we should add it to this pull request?